### PR TITLE
(improvement) backtest: use tradingview chart

### PR DESCRIPTION
**Pull request by bhoomij**

### CODE REVIEW

File: src/components/Backtester/reports/HistoricalReport.js [hunk 1]
```diff
@@ -61,26 +66,63 @@ const HistoricalReport = (opts, results, backtestData, backtestOptions, t, setti
...
+      //       width={width}
+      //       height={height}
+      //       // onAddIndicator={onAddIndicator}
+      //       // onDeleteIndicator={onDeleteIndicator}
+      //       isSyncing={false}
+      //       candleLoadingThreshold={3} // we always get 1 candle when sub'ing
+      //       // bgColor={chartColours.bgColor}
+      //       // config={chartColours}
+      //       // candleWidth={tf}
+      //       disableToolbar
```
Comment (bhoomij) [inline src/components/Backtester/reports/HistoricalReport.js:142]: keeping older chart version commented for sometime to easily switch to older chart for debugging or comparing features with tv-chart

File: src/components/Backtester/reports/HistoricalReport.js [hunk 2]
```diff
@@ -61,26 +87,63 @@ const HistoricalReport = (opts, results, backtestData, backtestOptions, t, setti
...
+      //       // onAddIndicator={onAddIndicator}
+      //       // onDeleteIndicator={onDeleteIndicator}
+      //       isSyncing={false}
+      //       candleLoadingThreshold={3} // we always get 1 candle when sub'ing
+      //       bgColor={chartColours.bgColor}
+      //       config={chartColours}
+      //       candleWidth={tf}
+      //       disableToolbar
+      //       showMarketLabel={false}
+      //     />
```
Comment (bhoomij) [inline src/components/Backtester/reports/HistoricalReport.js:144]: keeping older chart code as commented for sometime to easily switch for debugging and compare features with tv-chart

**Review by bhoomij — COMMENTED** [review]
keeping older chart code commented for sometime

File: src/components/Backtester/reports/HistoricalReport.js [hunk 3]
```diff
@@ -1,53 +1,79 @@
+// eslint-disable
```
Comment (Blaumaus) [inline src/components/Backtester/reports/HistoricalReport.js:1]: Maybe we should only disable some specific ESLint rules, but not the ESLint itself here?

**Review by Blaumaus — APPROVED** [review]

File: src/components/Backtester/reports/HistoricalReport.js (hunk 3 above)
Comment (bhoomij) [reply src/components/Backtester/reports/HistoricalReport.js]: removed as it was added for local testing purpose only

**Pull request merged**